### PR TITLE
Hostname instead IP in kafka-proxy-sidekick.

### PR DESCRIPTION
**Pull request by kovacshuni**

### CODE REVIEW

File: kafka-rest-proxy-sidekick.service [hunk 1]
```diff
@@ -5,12 +5,12 @@ After=kafka-rest-proxy.service
 [Service]
 ExecStartPre=/bin/sh -c "\
   etcdctl set /vulcand/backends/kafka-rest-proxy/backend '{\"Type\": \"http\"}' ; \
-  etcdctl set /vulcand/frontends/kafka-rest-proxy/frontend '{\"Type\": \"http\", \"BackendId\": \"kafka-rest-proxy\", \"Route\": \"PathRegexp(`/.*`) && Host(`kafka`)\"}';" 
+  etcdctl set /vulcand/frontends/kafka-rest-proxy/frontend '{\"Type\": \"http\", \"BackendId\": \"kafka-rest-proxy\", \"Route\": \"PathRegexp(`/.*`) && Host(`kafka`)\"}';"
 
 ExecStart=/bin/sh -c "\
-  export KAFKA_PROXY_IP=$(/usr/bin/ifconfig eth0 | sed -n '2 p' | awk '{print $2}' | cut -d':' -f2); \
+  export KAFKA_PROXY_IP=$(localhost); \
```
Comment (catalin-ilea) [inline kafka-rest-proxy-sidekick.service]: why localhost?

File: kafka-rest-proxy-sidekick.service [hunk 2]
```diff
@@ -5,12 +5,12 @@ After=kafka-rest-proxy.service
...
   while true; do \
-    export KAFKA_PROXY_PORT=$(/usr/bin/docker port kafka-rest-proxy 8082 | cut -d':' -f2); \
+    export KAFKA_PROXY_PORT=$(localhost); \
```
Comment (catalin-ilea) [inline kafka-rest-proxy-sidekick.service]: and here? this is a port

**Comment by catalin-ilea** [comment]
+1

**Comment by kovacshuni** [comment]
Changed all services now.

**Pull request merged**